pom.xml

```diff
@@ -29,8 +30,13 @@
     </site>
   </distributionManagement>
   <properties>
+    <karaf-maven-plugin.aggregateFeatures>false</karaf-maven-plugin.aggregateFeatures>
+    <karaf-maven-plugin.version>3.0.8</karaf-maven-plugin.version>
     <mockito.version>1.9.5</mockito.version>
     <guava.version>17.0</guava.version>
+    <commons.collections.version>3.2.2</commons.collections.version>
+    <mongo.java.driver.version>3.2.2</mongo.java.driver.version>
+    <pentaho-modeler.version>8.1.0.0-SNAPSHOT</pentaho-modeler.version>
```

Comment:
Use `${project.version}`instead of `8.1.0.0-SNAPSHOT`

Reply:
Check!